Sage/carbon
# feat(step-sequence): describe step sequence using design tokens, update tests

### Proposed behaviour

Describes step sequence using the tokens of the design.
Changes the height of the sequence progress indicator: `var(--sizing025)` ( 2px ).
Updates tests to include changes.

### Current behaviour

Step sequence component use the theme styled-component property.
Current height of the sequence progress indicator is 1px.

### Checklist




#### QA


### Additional context

![image](https://user-images.githubusercontent.com/44143630/150525540-932c219c-886b-458e-a1db-9fa9b62625fd.png)

### Testing instructions

The correct appearance can be checked in the component: Step sequence.
Use devtools to check if anchor navigation use css variables.
